include/indexstore/indexstore.h

```diff
@@ -149,8 +155,8 @@ indexstore_store_set_unit_event_handler(indexstore_t,
 
 INDEXSTORE_PUBLIC void
 indexstore_store_set_unit_event_handler_f(indexstore_t, void *context,
-            void(*handler)(void *context, indexstore_unit_event_notification_t),
-                                          void(*finalizer)(void *context));
+            INDEXSTORE_NOESCAPE void(*handler)(void *context, indexstore_unit_event_notification_t),
```

Comment:
This one is incorrect: the unit event handler is an async callback.

Reply:
Thanks for catching this. I did see that the event handling was async, and left it out, except I missed this.

Reply:
fixed